src/views/Project/index.jsx

```diff
@@ -0,0 +1,37 @@
+import { Component } from 'react';
+import Typography from 'material-ui/Typography';
+import ReactMarkdown from 'react-markdown';
+import projects from '../../data/loader';
+import BugsTable from '../../components/BugsTable';
+
+class Project extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      fileName: props.match.params.projectKey,
+      projects,
+    };
+  }
+  render() {
+    const { fileName, projects } = this.state;
+    const projectInfo = projects[fileName];
+
+    return (
+      <div>
+        <header>
+          <Typography variant="display1" align="center">
+            {projectInfo.name}
+          </Typography>
+          {projectInfo.description && (
```

Comment:
I think we should display the project description in `/projects` or in `/projects/:project` (not both). I'm leaning towards the former. You can remove it from this page. Feel free to object.

Reply:
I would add "Bugs & Issues" here.

```jsx
<Typography variant="subheading" align="center">
  Bugs & Issues
</Typography>
```

Reply:
But isn't there already Bugs and Issues on top of the table? 
I feel like moving it might make the page too empty, and since I initially plan the description to be description of the project, I plan to only display the first few words in the /projects and all  of them in /projects/:project what do you think?